### DISCUSSION TITLE
AlphaTab logo

### DISCUSSION THREAD

**Discussion by bali182** [CoderLine/alphaTab#1608] [General]
I've made a little project using AlphaTab. 

For now I removed the "rendered by alphaTab" text from the SVG itself, and "moved" it to corner of the page, in the form of a logo-like element (also made it bigger and clickable, I definitely don't want to hide that it was rendered by AlphaTab :).

If it's ok with you, I'd like to make an SVG version of the logo that you are using on the website, that is scalable and can easily be embedded into anything that wants to showcase AlphaTab.

Do you remember what font was used to create the website logo? I could try to just trace it in InkScape, but website logo is so small that I'd lose a lot of detail.

![Screenshot 2024-08-09 105727](https://github.com/user-attachments/assets/d8fc516d-ab38-4695-9b3c-c9198b341454)

**Comment by Danielku15** [comment]
This kind of attribution is fine for me, thanks a lot for respecting it that carefully 😄 
No need to create SVGs yourself, I have some floating around. Feel free to tweak them to your needs. 

![logo-horizontal](https://github.com/user-attachments/assets/773614b3-cdbe-4682-9bb1-f500f66f568d)
![logo-vertical](https://github.com/user-attachments/assets/6e8ff50b-89ea-40f9-a4d6-a99bfd04458e)

**Reply by bali182** [reply]
Amazing thank you! :)

**Reply by LIUBINfighter** [reply]
<img width="2560" height="1599" alt="image" src="https://github.com/user-attachments/assets/597b4178-e004-4378-8182-952496468ba2" />

Looks pretty, thanks! I have not notice this discussion before 🤣.

**Comment by tbreuss** [comment]
First of all, thank you very much for your work and the excellent library @Danielku15! AlphaTab really is asome work!

I have a follow-up question on the subject.

Is it okay to remove “rendered by alphaTab” from the individual scores and mention AlphaTab on a central page with a link to the AlphaTab website?

And if a link is desired, should it be rel=follow or rel=nofollow?

PS. The main reason for my question is, that I'll have a lot of small score snippets and the "rendered by..." doesn't work well with such a setup.

**Reply by Danielku15** [reply]
I've sent you a mail to discuss the details for a change in attributing alphaTab. generally I judge and discuss this on a project-by-project basis.

**Reply by tbreuss** [reply]
I'll reply to you later via email.
